### PR TITLE
Backend-Ruby_TestFirst 2.3??

### PR DESCRIPTION
**Pull request by gardenqu**

### CODE REVIEW

**Comment by jefferson-neel** [comment]
I don't see all of the files here, only the one for example `06`. Were you able to complete the other exercises and they just didn't get committed/pushed? Wanted to double check before I review.

Post a comment on here if/when you push up the other exercises so I'll know to take another look.

Thanks!

**Comment by gardenqu** [comment]
I pushed it again

**Comment by jefferson-neel** [comment]
Perfect! I see them now, I'll review shortly :)

File: 02_calculator/calculator.rb [hunk 1]
```diff
@@ -1 +1,16 @@
...
+def add(a,b)
+    a+b
+end
+
+def subtract(a,b)
+    a-b
+end
+
+def sum(arr)
+    arr.sum
```
Comment (jefferson-neel) [inline 02_calculator/calculator.rb:12]: Great use of `.sum`! 👏

**Review by jefferson-neel — APPROVED** [review]
This all looks really good! Great job! 👏

**Pull request merged**